vscode/package.json

```
@@ -661,6 +651,16 @@
         "key": "shift+alt+/",
         "when": "cody.activated && editorTextFocus && editorHasSelection"
       },
+      {
+        "command": "cody.chat.new",
+        "key": "shift+alt+l",
+        "when": "cody.activated && !editorTextFocus"
+      },
+      {
+        "command": "cody.chat.new",
+        "key": "shift+alt+/",
+        "when": "cody.activated && !editorTextFocus"
+      },
```

Does it make sense to leave the existing keybindings in place for Linux and MacOS users and only introduce the new bindings for windows users? Or do we not think people are using these?

```suggestion
      {
        "command": "cody.chat.new",
        "key": "shift+ctrl+l",
        "when": "cody.activated && !editorTextFocus && !isWindows"
      },
      {
        "command": "cody.chat.new",
        "key": "shift+ctrl+/",
        "when": "cody.activated && !editorTextFocus && !isWindows"
      },
      {
        "command": "cody.chat.new",
        "key": "shift+alt+l",
        "when": "cody.activated && !editorTextFocus && isWindows"
      },
      {
        "command": "cody.chat.new",
        "key": "shift+alt+/",
        "when": "cody.activated && !editorTextFocus && isWindows"
      },
```